spec/spec_helper.rb

```diff
@@ -77,11 +77,19 @@
 end
 
 require 'support/active_record'
+require 'ovirt_metrics'
+
+OvirtMetrics.config do |c|
+  # Necessary because you currently cannot specify the
+  # connection name for ActiveRecord::Schema, which we use
+  # in tests to reset db schema for different RHEV versions.
+  c.connection_specification_name = 'primary' if ActiveRecord::VERSION::MAJOR >= 5
+end
+ActiveRecord::Base.establish_connection :adapter => "sqlite3", :database => ":memory:"
```

Comment:
Earliest possible time _after_ setup code and right before you run your tests, no?